src/delegation/Delegation.chain.ts

```diff
@@ -37,7 +37,10 @@ export async function getChildIds(
   id: IDelegationBaseNode['id']
 ): Promise<string[]> {
   const blockchain = await getCached()
-  return Blockchain.asArray(await blockchain.api.query.delegation.children(id))
```

Comment:
This is out of the scope of the ticket.

Reply:
I wouldn't think it is - it's a necessary change to get rid of the return type `any[]` of `Blockchain.asArray()` 
What we probably could do instead is convert each element to string in order to match the `string[]` return type of `getChildIds()`